### PR TITLE
added a check to compare the objects in common knowledge and labels as Janno requested

### PR DESCRIPTION
**Pull request by Jip238**
…s Janno requested

### CODE REVIEW

File: test_tools/data_collection/analyse-annotations [hunk 1]
```diff
@@ -70,9 +70,28 @@ if __name__ == "__main__":
...
+
+    print "objects that are present in objects_set but not in tensorflow_set\n"
+    for o in objects_set.difference(tensorflow_set):
+        print o
+    print "\n"
+
+    print "objects that are present in tensorflow_set but not in objects_set\n"
+    for t in tensorflow_set.difference(objects_set):
+        print t
+    print "\n"
```
Comment (LoyVanBeek) [inline test_tools/data_collection/analyse-annotations]:
```python
print
``` 

also prints a newline

Comment (MatthijsBurgh) [inline test_tools/data_collection/analyse-annotations]: So remove all the "\"??

Comment (LoyVanBeek) [inline test_tools/data_collection/analyse-annotations]:
```python
print "\n"
```
can be replaced with a simple 

```python
print
```

**Comment by LoyVanBeek** [comment]
I approved but then I noticed the extension to the file is missing. No biggie, but I'd add it for consistency

**Comment by MatthijsBurgh** [comment]
We have files with and without extension. Do we have any policy for that?

**Comment by LoyVanBeek** [comment]
We don't have any policies or coding style guides (besides Python PEP-8) etc, but that's why we have reviews.

**Pull request merged**